### DISCUSSION TITLE
Using acrylic option in microsoft terminal

### DISCUSSION THREAD

**Discussion by JasperSurmont** [JanDeDobbeleer/oh-my-posh#250] [Troubleshoot] [answered]
### Description

I don't really think this is possible, but I guess I'll ask anyway. The first icon of the powerline option will be transparent if there is no previous segment. In microsoft terminal you can use the acrylic option to make your terminal less opaque (more transparent). However, the powerline icon doesn't take this into account and will still display the background color as if it was fully opaque. (example can be seen [here](https://imgur.com/a/3mBT8py)). Is there any way to make the powerline icon apply the color after the acrylic has been applied?

### Environment

- Oh my Posh version: 3.57.0
- Theme: /
- Operating System: Windows
- Shell: Powershell
- Terminal: Microsoft terminal

**Comment by aessing** [comment]
Thanks @JasperSurmont, have the exact same issue.

**Comment by JanDeDobbeleer** [comment, marked as answer]
@JasperSurmont this is an issue in Windows Terminal, it's on [their](https://github.com/microsoft/terminal/issues/7014) backlog.
